### ISSUE TITLE
Compatibility with Java 17 ?

### ISSUE THREAD

**Issue by Blanbec1973** [tginsberg/junit5-system-exit#15]
Hi Todd,

I am encountering an error while using Juinit5-system-exit in Java 17 context : 
WARNING: A terminally deprecated method in java.lang.System has been called
WARNING: System::setSecurityManager has been called by com.ginsberg.junit.exit.SystemExitExtension (file:/C:/Users/heynerr/.m2/repository/com/ginsberg/junit5-system-exit/1.1.2/junit5-system-exit-1.1.2.jar)
WARNING: Please consider reporting this to the maintainers of com.ginsberg.junit.exit.SystemExitExtension
WARNING: System::setSecurityManager will be removed in a future release

Are you planning a new version soon that includes this version ?
Thank and regards !
R.

**Comment by tginsberg** [comment]
Hi @Blanbec1973 - This is a duplicate of #10. Check out that issue for an ongoing discussion.

TL;DR: There is no way to avoid this until Java has a new way to veto the JVM shutdown that the SecurityManager provides currently.

**Comment by tginsberg** [comment]
Duplicate #10

**Comment by Blanbec1973** [comment]
@tginsberg thank you for this quick answer and sorry for the duplicate !

**Comment by tginsberg** [comment]
Absolutely not a problem @Blanbec1973! I'd rather spend the time to mark something as a duplicate than not know about a problem people are having. I appreciate you taking the time to file an issue!
